Realm/ObjectStore/object_store.cpp

```diff
@@ -479,3 +479,17 @@ void ObjectStore::delete_data_for_object(Group *group, const StringData &object_
     }
 }
 
+bool ObjectStore::is_empty(Group *group) {
+    Schema schema = schema_from_group(group);
```

Comment:
That was my first pass, but didn't think it would be wise to duplicate this logic. Considering that `schema_from_group` didn't do much more, I wouldn't expect it to be much more expensive, but I'll happily extract that out.